src/pngchunk_int.cpp

```diff
@@ -165,22 +165,28 @@ namespace Exiv2 {
         else if(type == iTXt_Chunk)
         {
             const int nullSeparators = std::count(&data.pData_[keysize+3], &data.pData_[data.size_-1], '\0');
-            enforce(nullSeparators >= 2, Exiv2::kerCorruptedMetadata);
+            enforce(nullSeparators >= 2, Exiv2::kerCorruptedMetadata, "iTXt chunk: not enough null separators");
```

Comment:
The message you added will not be displayed, as `kerCorruptedMetadata` does not include it in the output via `e.what()`.

Reply:
I noticed that the message was not printed, but I thought it could be a good idea to have that string there in case we can add unit tests in the future around this code. I did not check though if we could get the string when catching the exception in the unit tests code.

Anyways, I think it does not hurt to have a small string in the `enforce`calls giving some details of why we are placing the enforce calls in the code.

Reply:
It will hurt the performance slightly in case the exception gets thrown, as then the `Exiv2::Error` constructor will search for a place where to insert the string but won't be able to find one. Not really a big deal though. In case you want the error message printed, we you can use `kerErrorMessage` (albeit that is a quite generic error).